lib/active_model/serializer.rb

```diff
@@ -28,8 +28,8 @@ def self.inherited(base)
       base._attributes_keys = self._attributes_keys.try(:dup) || {}
       base._associations = self._associations.try(:dup) || {}
       base._urls = []
-      serializer_file = File.open(caller.first[/^[^:]+/])
-      base._cache_digest = Digest::MD5.hexdigest(serializer_file.read)
+      serializer_file_path = caller.first[/\A\S+(?=:\d+:in)/]
+      base._cache_digest = Digest::MD5.hexdigest(File.read(serializer_file_path))
     end
```

Comment:
OK, be seriously, there is already (testing)[https://github.com/Eric-Guo/active_model_serializers/blob/151301c8b7b0c61cd138434472e418c032ca433b/test/fixtures/poro.rb#L50] existing, but the problem is if you not running in Windows, c:/git won't return to you from ruby side, so it's really hard to write a tester to testing that case in Linux.